spanner/google/cloud/spanner_v1/database.py

```diff
@@ -264,6 +264,8 @@ def update_ddl(self, ddl_statements):
 
         :type ddl_statements: Sequence[str]
         :param ddl_statements: a list of DDL statements to use on this database
+        :type operation_id: str
+        :param operation_id: a string ID for the long-running operation
```

Comment:
Please add `(optional)` at the start of the description.

Reply:
Done.